ldms/src/decomp/static/decomp_static.c

```diff
@@ -452,7 +453,9 @@ __decomp_static_config( ldmsd_strgp_t strgp, json_entity_t jcfg,
 	i = 0;
 	TAILQ_FOREACH(jrow, &jrows->item_list, item_entry) {
 		if (jrow->type != JSON_DICT_VALUE) {
-			DECOMP_ERR(reqc, EINVAL, "a row must be an object\n");
+			DECOMP_ERR(reqc, EINVAL, "strgp '%s': row '%d': "
```

Comment:
Again maybe say "the item list must contain dictionaries." which means more to  someone writing JSON.